google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -252,8 +252,8 @@ def self.default_project
         #
         def query_job query, params: nil, priority: "INTERACTIVE", cache: true,
                       table: nil, create: nil, write: nil, large_results: nil,
-                      flatten: nil, dataset: nil, legacy_sql: nil,
-                      standard_sql: nil
+                      flatten: nil, dataset: nil, standard_sql: nil,
```

Comment:
Gotcha. Can we at the make resolve_legacy_sql place standard_sql at a high precedence than legacy_sql since standard_sql has prominence over legacy_sql.